tools/report_build.py

```diff
@@ -48,7 +49,10 @@ class GitHubQLQuery(GraphQLQuery):
 
 class GetEvents(GitHubQLQuery):
     query = """
-        query GetEvents($owner: String!, $name: String!, $pull_request: Int!, $count: Int!) {
+        query GetEvents($owner: String!,
+                        $name: String!,
+                        $pull_request: Int!,
+                        $count: Int!) {
```

Comment:
I wish our clang-format accepted this sort of formatting in C++, it often makes for much more readable code.